hivemq-edge/build.gradle.kts

```diff
@@ -503,7 +491,6 @@ downloadLicenses {
     )
 
     dependencyConfiguration = "runtimeClasspath"
-    excludeDependencies = listOf("com.hivemq:hivemq-extension-sdk:${property("hivemq-extension-sdk.version")}")
```

Comment:
why was this removed?

Reply:
it did not work without the version and having the version only for this sake seems overkill